client/src/locales/en-US.json

```
@@ -551,24 +551,24 @@
                 "cancelGreet": "Cancel"
             },
             "subtitles": {
-                "provideHostCode": "Click the code below on the guest.",
-                "getGuestCode": "Click on the code that appears on the guest.",
+                "provideHostCode": "Click the code below on the guest device.",
+                "getGuestCode": "Click on the code that appears on the guest device.",
                 "summary": "The device {label} has been added to your profile.",
-                "getDeviceInfo": "Waiting for you to fill the information on the guest.",
-                "waitForGuest": "You need to have Parsec open on both devices for the inscription duration.",
+                "getDeviceInfo": "Waiting for you to fill the information on the guest device.",
+                "waitForGuest": "You must have Parsec open on both devices for the inscription duration.",
                 "scanOrShare1": "Scan the QR Code ",
                 "scanOrShare2": "or ",
                 "scanOrShare3": "share the link",
                 "cancelGreet": "Are you sure you want to cancel the greeting process?",
                 "copiedToClipboard": "Copied to clipboard"
             },
             "errors": {
-                "retrieveDeviceInfoFailed": "Could not retrieve the information.",
+                "retrieveDeviceInfoFailed": "Failed to retrieve the information.",
                 "unexpected": "An unexpected error occured",
-                "createDeviceFailed": "Could not create the device.",
+                "createDeviceFailed": "Failed to create the device.",
                 "startFailed": "Failed to start the process. Please make sure that you are connected to the internet.",
-                "invalidCodeSelected": "You didn't select the correct code. Please restart the process.",
-                "noneCodeSelected": "If you didn't see the correct code, it could be a security concern. Please restart the process.",
+                "invalidCodeSelected": "You did not select the correct code. Please restart the process.",
+                "noneCodeSelected": "If you did not see the correct code, it could be a security concern. Please restart the process.",
```

Here I would use the same term as above ("onboarding" or whatever).
Also, I'd add that the "security concern" is related to the current process, that's why we ask to restart it.
```suggestion
                "invalidCodeSelected": "You did not select the correct code. Please restart the onboarding process.",
                "noneCodeSelected": "If you did not see the correct code, this could be a sign of a security issue during the process. Please restart the onboarding process.",
```